UI/GameInfoCache.cpp

```diff
@@ -375,6 +383,7 @@ class GameInfoWorkItem : public PrioritizedWorkQueueItem {
 						goto handleELF;
 					}
 					ERROR_LOG(LOADER, "invalid pbp %s\n", pbpLoader->Path().c_str());
+					info_->DisposeFileLoader();
```

Comment:
My memory tells me that this happens eventually anyway, although I saw you add it to the destructor so I might be wrong?  Is this leaking?

DisposeFileLoader had originally been added, iirc, to avoid holding on to too many concurrent file descriptors.

-[Unknown]